cmd/gindoid/web.go

```diff
@@ -93,7 +93,8 @@ func startDOIRegistration(w http.ResponseWriter, r *http.Request, jobQueue chan
 	}
 	// TODO Error checking
 	uuid := makeUUID(dReq.Repository)
-	doiInfo, err := parseDOIInfo(dReq.Repository, conf)
+	infoyml, err := readFileAtURL(dataciteURL(dReq.Repository, conf))
```

Comment:
The `err` here is ignored and overwritten. Since the todo comment above is about error checking, I assume this is known and will be changed later?